tests/test_client.py

```diff
@@ -28,13 +28,14 @@
     ENCODING_UTF_8,
     DOCUMENT_IMPORTER_TEXT,
 )
+import os
 
 URL_BASE = "http://localhost:8080"
 API_BASE = URL_BASE + "/rest/v1"
 TEST_API_TOKEN = "I-am-a-dummy-API-token"
 
 logging.basicConfig(level=logging.INFO)
-
+TEST_DIRECTORY = os.path.dirname(__file__)
```

Comment:
If I read it correctly, then `__file__` resolves to the module that is importing another module. So its value depends on who is importing this file. Why does it (always?) resolve to a file in the project root and thus can be used to construct the path to the test resources?

Reply:
Well, seems to be a best practice... fine